tests/tasks/configs/test_loop_config.py

```diff
@@ -80,8 +81,14 @@ def test_loop_config_with_subtask(task_workbench, windows, dialog_sleep):
     process_app_events()
     sleep(dialog_sleep)
 
-    config.subconfig.task_name = ''
-    assert not config.ready
+    def dummy(self):
+        self.ready = False
+
+    monkeypatch.setattr(type(config.subconfig), 'check_parameters',
```

Comment:
I want to be sure that config will not be ready if subconfig is not but as for standard config it cannot happen I patch on the fly the method of the subconfig class. monkeypacth simply take care of undoing everything at the end of the test.